docs/src/examples/components/Popup/Usage/PopupExampleOnMultiple.shorthand.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
+import { Button, Popup } from '@stardust-ui/react'
+
+const PopupExampleOnMultiple = () => (
+  <div>
+    <Popup
+      trigger={<Button icon="expand" content="Click + Focus" aria-label="Click or focus button" />}
+      content="Hello from popup on click!"
+      on={['click', 'focus']}
```

Comment:
This doesn't work as expected.  Clicking the button makes the popup appear for a short interval and close.  This is because the click invokes an open then the element receives focus which immediately invokes a close.  

![http://g.recordit.co/hxF1Vt0hb4.gif](http://g.recordit.co/hxF1Vt0hb4.gif)

Reply:
Thanks for the catch. The problem was that the focus was opening the popup always, but should be only if it comes from the keyboard. It is fixed now.
![popup-fix](https://user-images.githubusercontent.com/4512430/50819479-6e348280-132a-11e9-8c1f-e7d0e914a18e.gif)